<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Button_detection">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>开始检测</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_showImg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>结果展示</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_readImg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>选择图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_saveImg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>保存结果</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_7E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="action_D"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_A"/>
    <addaction name="action_bug"/>
    <addaction name="action_A_2"/>
   </widget>
   <addaction name="menu_7E"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_A">
   <property name="text">
    <string>使用说明(&amp;U)</string>
   </property>
  </action>
  <action name="action_bug">
   <property name="text">
    <string>报告bug</string>
   </property>
  </action>
  <action name="action_A_2">
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
  </action>
  <action name="action_D">
   <property name="text">
    <string>删除(&amp;D)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
